### ISSUE TITLE
Conda update does not find newer version than 2.3.8

### ISSUE THREAD

**Issue by WDee** [spyder-ide/spyder#4360]
Dear all, 

I am currently running Spyder 2.3.8 on Python 2.7.12 64bits, Qt 4.8.7, PyQt4 (API v2) 4.11.4 on Windows. I normally launch Spyder from my desktop shortcut.

I am trying to upgrade my version of python, and have run: 

conda update conda (indicates version 4.3.16)
conda update qt pyqt (upgraded to 4..8.7-vc9_6)
conda update spyder 

however - conda update spyder indicates that the latest version it can find is 2.3.8, rather than 3.x.x

Apologies if i am doing something wrong.

What should I do to obtain the latest version? 

All the best,
W

## Dependencies

IPython >=1.0      :  5.3.0 (OK)
jedi >=0.8.1;<0.9.0:  0.9.0 (NOK)
matplotlib >=1.0   :  1.5.1 (OK)
pandas >=0.13.1    :  0.18.1 (OK)
pep8 >=0.6         :  1.7.0 (OK)
pyflakes >=0.5.0   :  1.5.0 (OK)
pygments >=1.6     :  2.2.0 (OK)
pylint >=0.25      :  1.7.0 (OK)
qtconsole >=4.0    :  4.2.1 (OK)
rope >=0.9.2       :  0.9.4 (OK)
sphinx >=0.6.6     :  1.5.5 (OK)
sympy >=0.7.3      :  1.0 (OK)
zmq >=2.1.11       :  16.0.2 (OK)

**Comment by WDee** [comment]
Removed matplotlib, and with it seaborn, ggplot, basemap and scikit-image, but unfortunately does not make a difference.

It still shows:

# All requested packages already installed.
# packages in environment at C:\Users\W\Anaconda:
#
spyder                    2.3.8                    py27_1

**Comment by ccordoba12** [comment]
You need to remove the `spyder-app` package for the update to proceed correctly.

**Comment by jennirinker** [comment]
The issue persists in my version of Anaconda. Package spyder-app is not installed. I tried uninstalling matplotlib before upgrading spyder, and it did not help.

conda: 4.3.16
python: 2.7.13 in root, 3.5.2 in virtual environment
spyder: 3.1.3 (attempting to upgrade to 3.1.4)

I attempted to upgrade in both the root environment and my virtual environment, and neither attempt worked.

Here is the output from the upgrade attempt of the root environment:
```
(C:\Anaconda2) C:\Users\rink>conda update spyder
Fetching package metadata .............
Solving package specifications: .

# All requested packages already installed.
# packages in environment at C:\Anaconda2:
#
spyder                    3.1.3                    py27_0
```

**Comment by ccordoba12** [comment]
Conda packages are still not available for 3.1.4. Please be patient.

**Comment by jennirinker** [comment]
Ah. Makes sense -- it says on the release page that the conda packages might take a few days. Spyder was telling me on start-up that a new package was available, so I thought maybe this update bug was the issue.  I'll remember to ignore Spyder's notice and give it a few extra days next time.

Thanks for your hard work. I appreciate Anaconda beyond words.
